sos/report/plugins/vdo.py

```diff
@@ -28,6 +28,18 @@ def setup(self):
         vdos = self.collect_cmd_output('vdo list --all')
         for vdo in vdos['output'].splitlines():
             self.add_cmd_output(f"vdo status -n {vdo}")
-        self.add_cmd_output('vdostats --human-readable')
+        self.add_cmd_output([
+            'vdostats --human-readable',
+            'vdostats --verbose',
+        ])
+        vdo_cols = 'vdo_deduplication,vdo_index_state'
+        lvs_opts = '-a -o +'
```

Comment:
Why the two extra variables, if you use either of them just once?

Reply:
Two reasons:

- The LVS command was too long before I removed the headers option, so using these two variables I was within the limit, and
- The request has a very big list of columns and I'll need to split the output in at least two variables to make sure is readable. 

I'm still playing with the columns I need to display, so the name vdo_cols will probably change.

Reply:
OK, could be.. Bit redundant for the code, more readable for users.

Reply:
This is still a draft PR so I'll see how can I improve it